client/src/Infographic/Infographic.tsx

```diff
@@ -79,67 +83,64 @@ export const Infographic: FunctionComponent<InfographicData> = ({
     );
   }, [orgnr, miljø]);
 
-  const bransjeEllerNæring = sykefraværBransje ? "bransje" : "næring";
-
   return (
     <div className={styles.infographicWrapper}>
       <InfographicFlis
         ikon={<NorwegianFlag {...ikonstorrelse} />}
         tekst={"Sykefraværsprosenten i Norge det siste kvartalet er: "}
-        verdi={sykefraværNorge + "%"}
-        nedlastingPågår={nedlastingPågår}
+        verdi={(data.fraværsprosentNorge?.verdi ?? "- ") + "%"}
+        nedlastingPågår={data.nedlastingPågår}
       />
 
       <InfographicFlis
         ikon={<Bag {...ikonstorrelse} />}
-        tekst={`Sykefraværsprosenten i din ${bransjeEllerNæring} det siste kvartalet er: `}
-        verdi={(sykefraværBransje ?? sykefraværNæring) + "%"}
-        nedlastingPågår={nedlastingPågår}
+        tekst={`Sykefraværsprosenten i din ${prosenttypeBransjeEllerNæring} det siste kvartalet er: `}
+        verdi={(data.fraværsprosentBransjeEllerNæring?.verdi ?? "- ") + "%"}
+        nedlastingPågår={data.nedlastingPågår}
       />
 
       <InfographicFlis
         ikon={<HealthCase {...ikonstorrelse} />}
         tekst={"Vanligste årsak til sykemelding i Norge er: "}
         verdi={"Muskel- og skjelettplager"}
-        nedlastingPågår={nedlastingPågår}
+        nedlastingPågår={data.nedlastingPågår}
       />
 
       <InfographicFlis
         ikon={
           <Up
-            className={roterTrendpil(stigningstallTilTekst(trendStigningstall))}
+            className={roterTrendpil(data.trendBransjeEllerNæring?.verdi)}
             {...ikonstorrelse}
           />
         }
-        tekst={`Sykefraværet i din ${bransjeEllerNæring} de to siste kvartalene er `}
-        verdi={stigningstallTilTekst(trendStigningstall)}
-        nedlastingPågår={nedlastingPågår}
+        tekst={`Sykefraværet i din ${trendtypeBransjeEllerNæring} er: `}
```

Comment:
Jeg tror vi må jobbe litt med teksten her. Når `Trend` er tomt da får vi følgende tekst: "Sykefraværet i din næring er: -" 
Synes det sier lite om at denne infographic her handler om trend. Kanskje noe à la "Vi har ikke nok data for å beregne utviklingen av sykefraværsstatistikk for din bransje/næring ila et år"

Reply:
Enig med deg! Kan ta en kjapp prat med Monica og høre hva hun tenker om ny tekst

Reply:
Lagde ny oppgave på det